pkg/cli/config/config.go

```diff
@@ -70,3 +74,46 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 
 	return bpConfig, nil
 }
+
+// CheckAPIConnection validate API connection via configured proxy and VPN
+func (config BackplaneConfiguration) CheckAPIConnection() error {
+
+	// Check backplane Proxy URL
+	if config.ProxyURL == "" {
+		//Cannot connect to backplane API URL: Check if you need to use a proxy/VPN to access backplane
+		return errors.New("empty proxy url - check your backplane-cli configuration in $HOME/.config/backplane/config.json")
+	}
+
+	// Check proxy connection
+	connectionOk, err := config.pingBackplaneAPIviaProxy()
+
+	if !connectionOk {
+		return err
+	}
+
+	return nil
+}
+
+// pingBackplaneAPIviaProxy returns status of the the API connection
+func (config BackplaneConfiguration) pingBackplaneAPIviaProxy() (bool, error) {
```

Comment:
Can we rename the function name here? As the implementation is to send a normal http request, the `ping` here might be a bit confusing.

Reply:
Rename the function to `testHttpRequestToBackplaneAPI()`, but let me know if you have a better option  :)